### ISSUE TITLE
Accessibility issue with custom Navigation buttons

### ISSUE THREAD

**Issue by nhanvervio** [nolimits4web/swiper#7528]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/p/sandbox/swiper-navigation-forked-jzzmnl?file=%2Findex.html%3A9%2C7

### Bug description

When registering custom navigation buttons using [Parameters as Props](https://swiperjs.com/element#parameters-as-props), the buttons are rendered outside the Shadow DOM created by `swiper-container`.

This is the correct behavior, except these **outside** buttons now have the `aria-controls` attribute pointing to a `swiper-wrapper` element **inside** the Shadow DOM. This is marked as a Critical accessibility error by tools like Axe or Lighthouse as the **Id** cannot be queried from outside the Shadow DOM.

![image](https://github.com/nolimits4web/swiper/assets/139731719/c13d3951-44bc-4484-995d-13a14b3ccc31)

I'm not sure what the solution is without tearing down the Shadow DOM altogether. 
Proposals like [AOM's IDREF](https://wicg.github.io/aom/caniuse.html) are promising, but it hasn't been supported by any browsers just yet. Alternatively, I can use Javascript to get rid of the `aria-controls` attributes, but this is just bypassing the WCAG spec.

### Expected Behavior

No accessibility issues flagged by Accessibility tools.

### Actual Behavior

`aria-controls` on custom navigation buttons are flagged as invalid.

### Swiper version

11.1.1

### Platform/Target and Browser Versions

MacOS 14.4 Chrome 124.0.6367.119

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by nolimits4web** [comment]
t0ggles-create swiper

**Comment by zebsola** [comment]
Is there any update on this issue? I am trying to make my app fully accessible but this is creating a blocker.

**Comment by mdmontesinos** [comment]
I'm also facing this issue. This makes Swiper with custom navigation buttons not accesible

**Comment by mdmontesinos** [comment]
Please, is there any way to fix this issue? It's currently the only thing preventing me from having a 100 accesibility score on my site.

**Comment by nolimits4web** [comment]
Resolved by #8176: `aria-controls` was removed from next/prev buttons entirely, since ARIA id references can't cross Shadow DOM boundaries (the `aria-label` already conveys the buttons' purpose). Released in v12.1.4 and included in all v14 releases. For older versions, see the `a11y.id` workaround above.
